### PR TITLE
Miscellaneous doc cleanup

### PR DESCRIPTION
**Pull request by ahardin-rh**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] Optimization
- [ x] Documentation Update

## Description

Typo fixes and consistency updates to the doc set.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: docs/measurements.md [hunk 1]
```diff
@@ -69,12 +69,12 @@ Pod latency quantile sample:
 }
 ```
 
-Where quantileName matches with the pod conditions and can be:
+Where q`uantileName` matches with the pod conditions and can be:
```
Comment (smalleni) [inline docs/measurements.md:72]: Typo?

Comment (ahardin-rh) [reply docs/measurements.md]: Oops! Yes, fixed!

**Review by vishnuchalla — APPROVED** [review]
lgtm!

File: docs/index.md [hunk 2]
```diff
@@ -6,19 +6,19 @@
 
 Kube-burner is a Kubernetes performance and scale test orchestration toolset. It provides multi-faceted functionality, the most important of which are summarized below.
 
-- Create, delete and patch Kubernetes resources at scale.
+- Create, delete, and patch Kubernetes resources at scale.
```
Comment (rsevilla87) [inline docs/index.md:9]: extra comma?

**Pull request merged**